cfgov/tccp/jinja2/tccp/card.html

```diff
@@ -23,737 +26,892 @@
+                        }}
+                        {{ 'Variable' if card.cash_advance_fee_calculation
+                            and card.cash_advance_fee_dollars is none
+                            and card.cash_advance_fee_percentage is none
+                        }}
+                        {{ currency(0) if not card.cash_advance_fees }}
+                    </div>
+                    <div class="h4 u-mt0">
+                        Cash advance fee
+                        {{- ', see details' if card.cash_advance_fee_calculation }}
+                    </div>
+                </div>
+            </div>
+            {% if card.cash_advance_fee_calculation %}
+                <p>
+                    Cash advance fee details:
+                    {{ card.cash_advance_fee_calculation }}
+                </p>
+            {% endif %}
+            {% if card.cash_advance_apr_vary_by_credit_tier %}
+                <p>
+                    Your cash advance APR will be based on your credit score.
+                    You’ll only know your exact APR if you are approved for the
+                    card. Here are average cash advance APRs for different
+                    credit scores for this card:
+                </p>
+                <table class="o-table o-table--apr">
+                    <thead>
+                        <tr>
+                            <th>Credit score</th>
+                            <th>Median balance transfer APR</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {% for advance_apr in [
+                            card.advance_apr_poor,
+                            card.advance_apr_good,
+                            card.advance_apr_great
+                        ] %}
+                            <tr>
+                                <td>{{ credit_tiers[loop.index][1] }}</td>
+                                <td>{{ apr(advance_apr) }}</td>
+                            </tr>
+                        {% endfor %}
+                    </tbody>
+                </table>
+            {% elif card.advance_apr_median is not none
+                and card.advance_apr_min != card.advance_apr_max %}
+                <p>
+                    You’ll only know your exact cash advance APR if you are
+                    approved for the card. The median cash advance APR for
+                    this card is {{ apr(card.advance_apr_median) }}.
+                </p>
+            {% endif %}
+            {# TODO: Explain this better #}
+            <p>
+               The cash advance fee is charged
+               {{ ' for each cash advance transaction' if
+                    card.cash_advance_fee_for_each_transaction else
+                    ' once per billing cycle'
+                }}.
+            </p>
+        {% else %}
+            <p>This card does not offer cash advanaces.</p>
         {% endif %}
-    </dl>
+    </div>
+
 </div>
```

Comment:
Perhaps we'll get to this in the next iteration, but the page feels like it ends a little unexpectedly, especially on desktop.

|Desktop|Mobile|
|-|-|
|<img width="1235" alt="image" src="https://github.com/cfpb/consumerfinance.gov/assets/654645/5f8256f8-74e7-438e-8c4f-f0477946cc0d">|<img width="571" alt="image" src="https://github.com/cfpb/consumerfinance.gov/assets/654645/fec0beb8-827b-41d9-a265-f6ac19e33630">|